assets/queries/cloudFormation/acm_expired_certificates/query.rego

```diff
@@ -0,0 +1,14 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Status == "EXPIRED"
```

Comment:
Checking the reference for AWS::CertificateManager::Certificate, there is no field Status. I think we cannot check this query via resource. Please review the query